sesman/notes.txt

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Assuming that committing this file was intentional, can you please add a header, or change the file name to clarify that this file is documenting the SCP protocol messages.

Reply:
Agreed.